scripts/end_to_end_test/run_end_to_end_test.py

```diff
@@ -175,6 +175,22 @@ def start_node(self, index):
 
         time.sleep(0.5)
 
+    def restart_node(self, index):
+        print('Restarting Node {}...'.format(index))
+
+        self._nodes[index].stop()
+
+        # Optically remove db files when testing recovering from a genesis file
+        if False:
```

Comment:
Make an option?

Reply:
it's going to be useful to keep that there for a short while as I will have to add an e2e test for the custom genesis block etc.